main.go

```diff
@@ -262,9 +267,15 @@ func doMain(args []string) error {
 		return nil
 	}
 
+	config, err := LoadConfig(*runOptions.config)
+	if err != nil {
+		return err
+	}
+
 	// in any case we will need to have a device
 	env := NewEnvironment(new(osCalls))
 	device := NewDevice(env, new(osCalls), "/dev/mmcblk0p")
+	store := NewDirStore(*runOptions.dataStore)
```

Comment:
Can we move this under https://github.com/mendersoftware/mender/pull/68/files#diff-7ddfb3e035b42cd70649cc33393fe32cR292. You don't need this except when you are running as a daemon.